### ISSUE TITLE
Bug: Evolines Toxel

### ISSUE THREAD

**Issue by KyrumX** [KyrumX/python3-dutch-pokemon-fandom-scripts#14]
```
Traceback (most recent call last):
  File "D:/Code/Github/python3-dutch-pokemon-fandom-scripts/test.py", line 144, in <module>
    scraper_class.setup()
  File "D:\Code\Github\python3-dutch-pokemon-fandom-scripts\pokemon_dex_entries\bulbapedia\pokedex_entry_bulbapedia.py", line 31, in setup
    dex_entry = parser_base.build_pokedex_entry()
  File "D:\Code\Github\python3-dutch-pokemon-fandom-scripts\pokemon_dex_entries\abstract_pokedex_entry_parser.py", line 50, in build_pokedex_entry
    self.parse_pokemon_evo_line(),
  File "D:\Code\Github\python3-dutch-pokemon-fandom-scripts\pokemon_dex_entries\bulbapedia\pokedex_entry_parser_bulbapedia.py", line 201, in parse_pokemon_evo_line
    main_line = self._create_evo_line(raw_evo_lines[0])
  File "D:\Code\Github\python3-dutch-pokemon-fandom-scripts\pokemon_dex_entries\bulbapedia\pokedex_entry_parser_bulbapedia.py", line 117, in _create_evo_line
    ndex=re.findall(r'\d+', evo_dict["no2a"])[0],
KeyError: 'no2a'
```

**Comment by KyrumX** [comment]
This is due to Bulbapedia defining different forms of the same evolution as different evolutions.
Toxel evolves into Toxtricity, however Toxtricity  has 2 forms. Bulbapedia sees this as two different evolutions, but it's the same evolution, just different forms.

**Comment by KyrumX** [comment]
Fixed with #15
